rfhub2/cli/api_client.py

```diff
@@ -0,0 +1,79 @@
+from requests import session, post, get, delete, exceptions
+from typing import Dict, List
+
+from rfhub2.model import Collection, Keyword
+
+
+API_V1 = 'api/v1'
+TEST_COLLECTION = {"name": "healtcheck_collection", "type": "a", "version": "a", "scope": "a",
+                   "named_args": "a", "path": "a", "doc": "a", "doc_format": "a"}
+
+
+class Client(object):
+    """
+    API client with methods to populate rfhub2 application.
+    """
+    def __init__(self, app_url: str, user: str, password: str):
+        self.app_url = app_url
+        self.session = session()
+        self.api_url = f'{self.app_url}/{API_V1}'
+        self.session.auth = (user, password)
+        self.session.headers = {"Content-Type": "application/json", "accept": "application/json"}
+
+    def check_communication_with_app(self) -> None:
```

Comment:
I think that my comment got lost in one of the previous PRs. I think that this check is too much, with adding and deleting test data on each run. For me it is enough that application will raise exception about user being unauthenticated on first post request. The meaningful error message at that point will give user the same information without any additional logic and any significant waiting time. If you want we can keep basic connectivity check here - for this i would suggest using healthcheck endpoint instead.

Reply:
Ok, I will get rid of it.

Reply:
With Basic auth there isn't really a point, but if we would introduce different auth scheme (like LDAP), then the step for logging in should be added and then basic auth could also implement this for the sake of having unified logic. Actually adding this endpoint would be a good way of faiiling fast when user does not pass credentials and would not require handling further in the code. Let's add it, but as a next PR, leave it like this for now, ok?